### PR TITLE
[ticket/14205] Bump PHP requirement to PHP 5.4

### PR DESCRIPTION
**Pull request by Nicofuma**
https://tracker.phpbb.com/browse/PHPBB3-14205

PHPBB3-14205

### CODE REVIEW

**Comment by bantu** [comment]
Should be use 5.4.0 instead of 5.4 in text?

**Comment by Nicofuma** [comment]
updated

File: .travis.yml [hunk 1]
```diff
@@ -33,19 +33,18 @@ services:
...
+  - travis/check-sami-parse-errors.sh $DB $TRAVIS_PHP_VERSION $NOTESTS
+  - travis/check-image-icc-profiles.sh $DB $TRAVIS_PHP_VERSION $NOTESTS
+  - travis/check-executable-files.sh $DB $TRAVIS_PHP_VERSION $NOTESTS ./
   - sh -c "if [ '$SLOWTESTS' != '1' -a '$DB' = 'mysqli' ]; then phpBB/vendor/bin/phpunit tests/lint_test.php; fi"
-  - sh -c "if [ '$SLOWTESTS' != '1' ]; then phpBB/vendor/bin/phpunit --configuration travis/phpunit-$DB-travis.xml; fi"
+  - sh -c "if [ '$NOTESTS' != '1' -a '$SLOWTESTS' != '1' ]; then phpBB/vendor/bin/phpunit --configuration travis/phpunit-$DB-travis.xml; fi"
   - sh -c "if [ '$SLOWTESTS' = '1' ]; then phpBB/vendor/bin/phpunit --configuration travis/phpunit-$DB-travis.xml --group slow; fi"
-  - sh -c "if [ '$TRAVIS_PHP_VERSION' = '5.3' -a '$DB' = 'mysqli' -a '$TRAVIS_PULL_REQUEST' != 'false' ]; then git-tools/commit-msg-hook-range.sh origin/$TRAVIS_BRANCH..FETCH_HEAD; fi"
-
+  - sh -c "set -x;if [ '$NOTESTS' = '1' -a '$TRAVIS_PULL_REQUEST' != 'false' ]; then git-tools/commit-msg-hook-range.sh $TRAVIS_COMMIT_RANGE; fi"
```
Comment (bantu) [inline .travis.yml]: You can probably use "sh -xc" and remove the "set -x".

File: travis/ext-sniff.sh [hunk 2]
```diff
@@ -10,10 +10,9 @@
 #
 set -e
 set -x
- 
+
 DB=$1
 TRAVIS_PHP_VERSION=$2
-EXTNAME=$3
```
Comment (nickvergessen) [inline travis/ext-sniff.sh]: hu?

Comment (Nicofuma) [inline travis/ext-sniff.sh]: my mistake, fixed

**Pull request merged**